tensorflow_addons/image/transform_ops.py

```diff
@@ -291,3 +295,54 @@ def _image_projective_transform_grad(op, grad):
         output_shape=tf.shape(image_or_images)[1:3],
         interpolation=interpolation)
     return [output, None, None]
+
+
+@tf.function
+def rotate(images, angles, interpolation="NEAREST", name=None):
+    """Rotate image(s) counterclockwise by the passed angle(s) in radians.
+    Args:
```

Comment:
Please revise the docstring format, refer to https://github.com/tensorflow/addons/pull/67/files#diff-558784788fba4d9e50933bfda99b1ed3R110